entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -1,130 +1,128 @@
-ms.author: divega
-ms.date: 10/27/2016
+title: ASP.NET Core - New database with Visual Studio 2017 | Microsoft Docs
+author: rick-anderson
+ms.author: riande
+ms.author2: tdykstra
+ms.date: 04/07/2017
 ms.assetid: e153627f-f132-4c11-b13c-6c9a607addce
 ms.technology: entity-framework-core
 uid: core/get-started/aspnetcore/new-db
 ---
 
-# ASP.NET Core - New Database
+# ASP.NET Core - New database with Visual Studio 2017
 
-> [!NOTE]
-> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../../ef6/index.md).
+In this walkthrough, you will build an ASP.NET Core MVC application that performs basic data access using Entity Framework Core. You will use migrations to create the database from your model. See [Additional Resources](#additional-resources)for more Entity Framework Core tutorials.
 
-> [!IMPORTANT]
-> The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.0.0 no longer supports `project.json` or Visual Studio 2015. Everyone doing .NET Core development is encouraged to [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/) and [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+This tutorial requires [Visual Studio 2017](https://www.visualstudio.com/downloads/) and the [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1 or later.
 
-In this walkthrough, you will build an ASP.NET Core MVC application that performs basic data access using Entity Framework. You will use migrations to create the database from your model.
+Notes: 
 
-> [!TIP]
-> You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb) on GitHub.
+- You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb) on GitHub.
+- This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../../ef6/index.md).
+- The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1.x no longer supports `project.json` or Visual Studio 2015. We recommend you [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/). If you are using Visual Studio, we recommend you migrate to [Visual Studio 2017](https://www.visualstudio.com/downloads/).
 
-## Prerequisites
+## Create a new project in Visual Studio 2017
 
-The following prerequisites are needed to complete this walkthrough:
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/)
+* **File > New > Project**
+* From the left menu select **Installed > Templates > Visual C# > .NET Core**.
+* Select **ASP.NET Core Web Application (.NET Core)**.
+* Enter **EFGetStarted.AspNetCore.NewDb** for the name and click **OK**.
+* In the **New ASP.NET Core Web Application** dialog enter:
 
-## Create a new project
+  * Under **ASP.NET Core Templates 1.1** select the **Web Application**
```

Comment:
No, that's the default, nothing to select. We could say something about 1.1 being the default. If you'd like to go that route please suggest new wording.